src/ViewInstance.js

```diff
@@ -518,6 +518,8 @@ class ViewInstance extends React.Component {
       );
     }
 
+    const seriesStatement = instance.series.map(x => ({value: x}));
```

Comment:
Add spaces after and before curly brackets, please
```suggestion
    const seriesStatement = instance.series.map(x => ({ value: x }));
```